pygmt/tests/test_text.py

```diff
@@ -78,6 +78,16 @@ def test_text_input_single_filename():
     return fig
 
 
+@pytest.mark.mpl_image_compare
+def test_text_input_external_filename():
+    """
+    Run text by passing in an external filename to textfiles
```

Comment:
GMT calls `@filename` remote files.
```suggestion
def test_text_input_remote_file():
    """
    Run text by passing in an remote file to textfiles
```

Reply:
Ah yes, I'll change that. Coming up with names are hard :laughing: